libafl/Cargo.toml

```diff
@@ -24,6 +24,7 @@ tui_monitor = ["tui", "crossterm"] # enable TuiMonitor with crossterm
 cli = ["clap"]  # expose bolts::cli
 qemu_cli = ["cli"]
 frida_cli = ["cli"]
+afl_exec_sec = [] # calculate exec/sec like AFL
```

Comment:
Why a feature flag and not a setting in the monitor(s) btw? Could be easier to use